service/assignment.rest.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<resource xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/rest-api-2.1.xsd"
+          name="tutorial" displayName="Assignment_L2 REST API" version="1.0">
+    <resource name="Assignment_l2_s1" require-authentication="anonymous-all">
+        <method type="post">
+            <service name="tutorial.Assignment_L2.post#Orders"/>
+        </method>
+    </resource>
+    <resource name="Assignment_l2_s2" require-authentication="anonymous-all">
+        <method type="post">
+            <service name="tutorial.Assignment_L2.post#OrderEntity"/>
+        </method>
+    </resource>
+    <resource name="Assignment_l2_s3" require-authentication="anonymous-all">
+        <method type="get">
+            <service name="tutorial.Assignment_L2.get#AllOrder"/>
+        </method>
+    </resource>
+    <resource name="Assignment_l2_s4" require-authentication="anonymous-all">
+        <method type="get">
+            <service name="tutorial.Assignment_L2.get#OrderDetails"/>
```

Comment:
Please check to use conventions to get details for a resource for a particular id.